cmd/openshift/operator/kodata/openshift/00-prereconcile/openshift-pipelines-scc.yaml

```diff
@@ -32,6 +32,8 @@ seLinuxContext:
   type: MustRunAs
 supplementalGroups:
   type: RunAsAny
+seccompProfiles:
+- runtime/default
```

Comment:
with removal of `runAsUser` and `runAsGroup` **seccompProfiles** is not required for Triggers

But keeping this as it is if at all it require for Pipeline

Reply:
can we check and then do?

Reply:
also it is the pipelines scc, not required for pipeline installation, so dont know if it is required necessarily

Reply:
> can we check and then do?

For triggers i did verify and its not required

For pipeline not getting how to ensure may be will create a sample pipelinerun and see if works will remove it

Reply:
It doesn't effect to Pipeline when we don't add label

 pod-security.kubernetes.io/enforce=restricted --overwrite=true

But when we add label we get below error for Pipeline
```
$ tkn pr logs -f sum-three-pipeline-run

task first-add has failed: pods "sum-three-pipeline-run-first-add-pod" is forbidden: violates PodSecurity "restricted:latest": unrestricted capabilities (containers "prepare", "step-add" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or containers "prepare", "step-add" must set securityContext.runAsNonRoot=true), seccompProfile (pod or containers "prepare", "step-add" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")
pod for taskrun sum-three-pipeline-run-first-add not available yet
Tasks Completed: 1 (Failed: 1, Cancelled 0), Skipped: 1

```

when we add 
```
seccompProfile:
   type: RuntimeDefault
```

we overcome seccomprofile error but still we have

```
$ tkn pr logs -f sum-three-pipeline-run

task first-add has failed: pods "sum-three-pipeline-run-first-add-pod" is forbidden: violates PodSecurity "restricted:latest": unrestricted capabilities (container "step-add" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "step-add" must set securityContext.runAsNonRoot=true)
pod for taskrun sum-three-pipeline-run-first-add not available yet
Tasks Completed: 1 (Failed: 1, Cancelled 0), Skipped: 1

```

Reply:
Alright as per this https://issues.redhat.com/browse/OCPSTRAT-487 Openshift enforce 
```
pod-security.kubernetes.io/enforce=restricted
```
from 4.17 so we are still good for 4.16